Sources/spm-test-finder/main.swift

```diff
@@ -0,0 +1,48 @@
+#if os(OSX)
```

Comment:
I would call this `swiftpm-xctest-helper` or something... I would rather not use `spm` internally, and think it makes sense to include xctest in the name.